### PR TITLE
Setup for python and travis

### PR DESCRIPTION
**Pull request by lindycoder**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Internap.
```
Comment (shiny-octo-adventure) [inline setup.py]: s/2015/2016

Comment (godp1301) [inline setup.py]: s/2015/2016

Comment (lindycoder) [inline setup.py]: Fixed

**Comment by godp1301** [comment]
👍

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = config-probe
+url = https://github.com/internaphosting/python-config-probe
```
Comment (idjaw) [inline setup.cfg]: s/internaphosting/internap/

Comment (lindycoder) [inline setup.cfg]: Thanks, fixed

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = config-probe
+url = https://github.com/internap/python-config-probe
+author = Internap Hosting
+author-email = opensource@internap.com
+summary = Auto-discovery of configurations for easy inline use
+description-file =
+    README.md
+classifier =
```
Comment (idjaw) [inline setup.cfg]: Don't know if this is necessary for the pypi info, but you *could* add:

license = Apache-2

We have this in a couple of our existing projects too.

Comment (mat128) [inline setup.cfg:9]: +1

**Pull request merged**